### PR TITLE
Create CONTRIBUTING.md

### PR DESCRIPTION
**Pull request by brizental**

### CODE REVIEW

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -0,0 +1,48 @@
...
+Contributions are always welcome and they can take many forms: bug reports, 
+documentation changes, feature requests or pull requests.
+
+Before you dive into technical details, take a moment to read our 
+[code of conduct](https://github.com/mozilla/glean.js/blob/main/CODE_OF_CONDUCT.md).
+Any form of communication from chat rooms and mailing lists to code comments
+is expected to follow this code of conduct, so that all can enjoy working on Glean.js.
+
+Glean.js' developer document lives in the [`docs/`](https://github.com/mozilla/glean.js/tree/main/docs)
+folder in this repository and Glean.js' user documenting lives in [The Glean Book](https://mozilla.github.io/glean/book/index.html).
```
Comment (brizental) [inline CONTRIBUTING.md:14]:
```suggestion
folder in this repository and Glean.js' user documentation lives in [The Glean Book](https://mozilla.github.io/glean/book/index.html).
```

File: CONTRIBUTING.md [hunk 2]
```diff
@@ -0,0 +1,48 @@
...
+
+If you are using Glean.js and found a bug or a missing feature, please file a bug.
+
+Glean.js bugs are filed in Bugzilla. Use [this link](https://bugzilla.mozilla.org/enter_bug.cgi?product=Data+Platform+and+Tools&component=Glean.js&priority=P4&status_whiteboard=%5Btelemetry%3Aglean-js%3Am%3F%5D) 
+to file a bug in the Glean.js Bugzilla component.
+
+## Finding a bug to work on
+
+As mentioned, Glean.js bugs are filed in Bugzilla. Anything on the Glean.js component
+with the tag [`[good first bug]`](https://bugzilla.mozilla.org/buglist.cgi?f1=status_whiteboard&v1=%5Bgood%20first%20bug%5D&o1=substring&resolution=---&query_format=advanced&f2=component&v2=Glean.js&list_id=15653400&o2=equals&classification=Client%20Software&classification=Developer%20Infrastructure&classification=Components&classification=Server%20Software&classification=Other)
```
Comment (Dexterp37) [inline CONTRIBUTING.md:31]: Mh, are we using tags? Mozilla uses the good-first-bug keyword AFAIK

Comment (brizental) [reply CONTRIBUTING.md:31]: Not sure, I copied the SDK here.

Comment (brizental) [reply CONTRIBUTING.md:31]: I mean "whiteboard tags" here, not sure that makes a difference.

**Review by Dexterp37 — APPROVED** [review]

**Pull request merged**